### PR TITLE
Feature/notification

### PR DESCRIPTION
**Pull request by psubham**

### CODE REVIEW

File: ng-components/src/app/demo/demo-notification/demo-notification.component.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+  get Badge() {
+    return JSON.stringify(this.demoNotification, null, 2);
+  }
+
+  set Badge(v) {
```
[the remark this answers is not in the document]
Comment (psubham) [reply ng-components/src/app/demo/demo-notification/demo-notification.component.ts]: change the setter

File: ng-components/src/app/demo/demo-notification/demo-notification.component.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    header: 'Notification Title',
+    // tslint:disable-next-line: max-line-length
+    content: 'If needed (e.g. error message notifications), add extra information about the specific nature of a problem and a way to solve it.'
+  };
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+  get Badge() {
```
[the remark this answers is not in the document]
Comment (psubham) [reply ng-components/src/app/demo/demo-notification/demo-notification.component.ts]: change the name

File: ng-components/src/app/core/notification/notification.component.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import { Component, OnInit, Input, Directive } from '@angular/core';
+
+@Directive({
+    // tslint:disable-next-line: directive-selector
```
[the remark this answers is not in the document]
Comment (psubham) [reply ng-components/src/app/core/notification/notification.component.ts]: Ok

File: ng-components/src/app/demo/demo-notification/demo-notification.component.scss [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+      display:flex;
+      flex-direction: row;
+      justify-content: space-between;
+  }
+  
+  .output-section-button{
+    display:inline-flex;
+   margin:9px;
+  }
+  .json{
```
[the remark this answers is not in the document]
Comment (psubham) [reply ng-components/src/app/demo/demo-notification/demo-notification.component.scss]: you mean the json, now I am not using it so I have removed it.

[the remark this answers is not in the document]
Comment (psubham) [reply ng-components/src/app/demo/demo-notification/demo-notification.component.scss]: resolve the spaces

[the remark this answers is not in the document]
Comment (tipsanemonk) [reply ng-components/src/app/demo/demo-notification/demo-notification.component.scss]: Na the code organizing

**Pull request merged**